quickstart/armv7a9-zynq7000-zturn.md

```diff
@@ -0,0 +1,245 @@
+# Running system on `armv7a9-zynq7000-zturn` (Xilinx Zynq-7000, Zturn development board)
+
```

Comment:
Please add a link to this page in armv7a9-zynq7000.md and armv7a9-zynq7000-zedboard.md, armv7a9-zynq7000-qemu.md (See also).